docs/ports/index.md

```diff
@@ -10,6 +10,8 @@ sensor multiplexers fall into this category (see list below). Simple
 multiplexers that only support one connection type will not have a separate
 driver for the port.
 
+BrickPi Input Port 5 is RaspberryPi I2C physically wired to the port pins. See <a href="brickpi-in-port-5">brickpi-in-port-5</a> for the details. 
```

Comment:
> BrickPi Input Port 5 is RaspberryPi I2C physically wired to the port pins.

This sentence is not very clear. Maybe "Input Port 5 on the BrickPi is pyhsically wired to the I2C pins on the Raspberry Pi".

Also, markdown does not need html tags for the link. Replace `<a href="brickpi-in-port-5">brickpi-in-port-5</a>` with `[brickpi-in-port-5]` and also `[brickpi-in-port-5]: /docs/ports/brickpi-in-port-5` on a new line.